plugins/out_file/file.c

```diff
@@ -316,7 +325,9 @@ static int template_output(FILE *fp, struct flb_time *tm, msgpack_object *obj,
     if (inbrace) {
         fputs(inbrace, fp);
     }
-    fputs(NEWLINE, fp);
+    if (ctx->line_append == 0){
```

Comment:
Could you compare with `FLB_FALSE` ?

Reply:
OK, I will change it in next commit.
Forgive me for my unfamiliar about C language style.